Signal/src/ViewControllers/ConversationView/Cells/OWSGenericAttachmentView.m

```diff
@@ -116,25 +117,37 @@ - (void)createContents
     [contentView autoPinEdgeToSuperviewEdge:ALEdgeTop withInset:self.vMargin];
     [contentView autoPinEdgeToSuperviewEdge:ALEdgeBottom withInset:self.vMargin];
 
+    OWSBezierPathView *iconCircleView = [OWSBezierPathView new];
+    UIColor *iconColor
+        = (self.isIncoming ? [UIColor colorWithRGBHex:0x9e9e9e] : [self foregroundColorWithOpacity:0.15f]);
+    iconCircleView.configureShapeLayerBlock = ^(CAShapeLayer *_Nonnull layer, CGRect bounds) {
+        CGFloat radius = MIN(bounds.size.width, bounds.size.height) * 0.5f;
+        layer.path = [UIBezierPath bezierPathWithRoundedRect:bounds cornerRadius:radius].CGPath;
```

Comment:
TIOLI, but would `[UIBezierPath bezierPathWithOvalInRect:bounds].CGPath` save you a line?

Reply:
Better thanks.